modules/ROOT/pages/gathering-set-up-information.adoc

```diff
@@ -0,0 +1,54 @@
+= Gathering Setup Information for SSO
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the following sections to gather the information you'll need to set up SSO.
```

Comment:
The short description should summarize, not introduce, the topic. Can you provide a quick summary of the information needed?